borg/helpers.py

```diff
@@ -312,17 +312,50 @@ def load_excludes(fh):
     both line ends are ignored.
     """
     patterns = (line for line in (i.strip() for i in fh) if not line.startswith('#'))
-    return [parse_pattern(pattern) for pattern in patterns if pattern]
+    return [parse_exclude_pattern(pattern)
+            for pattern in patterns if pattern]
```

Comment:
Because exclude patterns are now a special case of general patterns. `parse_pattern()` is now used to parse both `--pattern` and `--exclude` file-patterns and therefore a small wrapper `parse_exclude_pattern()` was needed for the `--exclude` case. (Like `parse_inclexcl_pattern()` is used for `--pattern`.)